apps/hash-frontend/src/pages/shared/org-form.tsx

```diff
@@ -202,23 +206,63 @@ export const OrgForm = ({
           </span>
         </Box>
       </InputGroup>
+      {initialOrg && (
+        <InputGroup>
+          <Label
+            label="Description"
+            hint="Provide a brief description of your organization"
+            htmlFor="description"
+          />
+          <TextField
+            id="description"
+            sx={{ width: 400 }}
+            {...register("description", { required: false })}
+          />
+        </InputGroup>
+      )}
```

Comment:
Maybe we should add a `@todo` to make it possible to set the description/location on creation as well?

Reply:
These are not in the design for the create org screen ([here](https://www.figma.com/file/gydVGka9FjNEg9E2STwhi2/HASH?type=design&node-id=8022-264328&mode=design&t=8TUgf14VHztIChEf-4)) so I believe they are intentional omissions